README.md

```diff
@@ -1,23 +1,54 @@
-# Einstein For Developers (Beta) 
+<div align=center>
 
-![Einstein](https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-thumb.jpeg)
+# <img src="https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-thumb.jpeg" width="50"> Einstein for Developers (Beta)
 
-This repo will be the primary means of *direct* communication between our Beta users and the team behind the Einstein for Developers! We'll be monitoring both the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) and [Discussions](https://github.com/forcedotcom/Einstein-GPT-for-Developers/discussions) tabs here for questions, feedback requests & more. 
+</div>
 
-Oh! And you can also provide general feedback, request product enhancements, start discussions with other Einstein for Developers users or the product team, and share best practices, using the [Einstein for Developers Trailblazer Group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94V000000oRJs?tab=discussion&sort=LAST_MODIFIED_DATE_DESC).
+Einstein for Developers (Beta) is an AI-powered developer tool that’s available as an easy-to-install Visual Studio Code extension built using CodeGen, the secure, custom AI model from Salesforce. Head over to the [Einstein for Developers Documentation](https://developer.salesforce.com/tools/vscode/en/einstein/einstein-overview) to learn more.
 
-We also encourage you to give us feedback on the _generated code_ using 👍, 👎 and comments inside the app. 
+## Get Started
+
+Einstein for Developers is available in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-einstein-gpt) and the [Open VSX Registry](https://open-vsx.org/extension/salesforce/salesforcedx-einstein-gpt).
+
+Go to [Einstein for Developers Set Up Documentation](https://developer.salesforce.com/tools/vscode/en/einstein/einstein-setup) for setup instructions.
+
+## Disable Einstein for Developers
+Einstein for Developers is enabled by default. You can easily disable or uninstall the Einstein for Developers extension at any time. Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the **View: Show Extensions** command (⇧⌘X). Search for "Einstein for Developers" in installed extensions list, and then select **Disable** or **Uninstall**.
+
+
+## What can Einstein for Developers Do?
+
+Use Einstein for Developers, an AI-powered developer tool, to enhance developer productivity.
+
+* **Use the inline code auto-completion feature to get completions for Apex, Javascript, HTML and CSS code**
+
+<div align=center>
+<img src="https://github.com/forcedotcom/Einstein-GPT-for-Developers/blob/main/images/einstein-inline-autocomplete.gif" width="750">
+</div>
+  
+* **Generate Apex Unit Tests to quickly get to required code coverage**
```

Comment:
Can we update the "required code coverage" with the language we agreed on from the walkthrough this morning?